### PR TITLE
Create side navigation and footer button

### PR DESCRIPTION
**Pull request by Susanne1307**
- Added footer button, which still has to be worked into the footer

- Added side navigation with open issues:
    - animation (slide in from the left when clicking menu button)
    - list items should link to the corresponding pages
    - profile picture should be inserted above the menu list

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
```
Comment (phlgr) [inline client/public/index.html:13]: Import the font in your GlobalStyles!

File: client/src/assets/check.svg [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (phlgr) [inline client/src/assets/check.svg:1]: Try minifying!

**Comment by phlgr** [comment]
Please add the links to your changes the next time. I had to guess that you built your components in storybook. Apart from that great Pull-Request!

File: client/public/index.html (hunk 1 above)
Comment (Susanne1307) [reply client/public/index.html:13]: #13 I plan to 👍

File: client/src/assets/check.svg (hunk 2 above)
Comment (Susanne1307) [reply client/src/assets/check.svg:1]: #14 Same here

**Pull request merged**